frontend/src/main/components/Commons/CommonsTable.js

```diff
@@ -35,8 +35,6 @@ export default function CommonsTable({ commons, currentUser }) {
         navigate(`/leaderboard/${cell.row.values["commons.id"]}`)
     }
 
-    const downloadCallback = (cell) => window.location.href = `/api/commons/${cell.row.values["commons.id"]}/download?commonsId=${cell.row.values["commons.id"]}`;
-
```

Comment:
I'm confused why is download being reverted?

Reply:
Good catch.  I'm asking the team now

Reply:
Should these lines be restored?